ibmcloud/terraform/cluster/ansible/kube-playbook.yml

```diff
@@ -0,0 +1,270 @@
+      args:
+        executable: /bin/bash
+        creates: /usr/local/bin/containerd
+    
+    - name: Ensure /etc/containerd directory exists  specific to config
+      file:
+        path: /etc/containerd
+        state: directory
+
+    - name: Copy containerd config file
+      copy:
+        dest: /etc/containerd/config.toml
+        content: |
+          version = 2
+          root = "/var/lib/containerd" 
+          state = "/run/containerd"
+          oom_score = -999
+
+          [grpc]
+            address = "/run/containerd/containerd.sock"
+            uid = 0
+            gid = 0
+
+          [debug]
+            address = "/run/containerd/debug.sock"
+            uid = 0
+            gid = 0
+            level = "debug"
+
+          [plugins]
+            [plugins."io.containerd.runtime.v1.linux"]
+              shim_debug = true
+            [plugins."io.containerd.grpc.v1.cri"]
+              [plugins."io.containerd.grpc.v1.cri".containerd]
+                default_runtime_name = "runc"
+                [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+                  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+                    runtime_type = "io.containerd.runc.v2"
+                  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata]
+                    runtime_type = "io.containerd.kata.v2"
+                    cri_handler = "cc"
+
+    - name: Copy systemd unit file for containerd
+      copy:
+        dest: /etc/systemd/system/containerd.service
+        content: |
+          [Unit]
+          Description=containerd container runtime
+          Documentation=https://containerd.io
+          After=network.target
+
+          [Service]
+          ExecStartPre=-/sbin/modprobe overlay
+          ExecStart=/usr/local/bin/containerd --config /etc/containerd/config.toml --log-level debug
+          Delegate=yes
+          KillMode=process
+
+          [Install]
+          WantedBy=multi-user.target
+
+    - name: Start containerd service
+      systemd:
+        name: containerd.service
+        state: started
+    
+    - name: Copy systemd unit file for fix-resolvconf
+      copy:
+        dest: /etc/systemd/system/fix-resolvconf.service
+        content: |
+          [Unit]
+          Description=Make /etc/resolv.conf a regular file
+          After=systemd-networkd.service
+
+          [Service]
+          ExecStart=/bin/bash -c 'rm -f /etc/resolv.conf && cp /run/systemd/resolve/resolv.conf /etc/resolv.conf'
+          ExecStop= /bin/bash -c 'rm -f /etc/resolv.conf && ln -s ../run/systemd/resolve/stub-resolv.conf /etc/resolv.conf'
+          Type=oneshot
+          RemainAfterExit=yes
+
+          [Install]
+          WantedBy=multi-user.target
+
+    - name: Start fix-resolvconf service
+      systemd:
+        name: fix-resolvconf.service
+        state: started
+
+    - name: Install IBM Cloud CLI
```

Comment:
We should test if the playbooks work when the Install IBM Cloud CLI and Install IBM Cloud CLI plugins tasks are only in the kata playbook.

Reply:
Tested and successfully reran Terraform template.
Remove IBMCloud CLI and plugin install from Kube-playbook

Reply:
I've tested the new playbooks and they work with installation of the IBM Cloud CLI only being in the kata playbook